<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RaptorSettingPage</class>
    <widget class="QWidget" name="RaptorSettingPage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1088</width>
                <height>689</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>26</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>16777215</width>
                <height>16777215</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="_SettingPageLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="_Header" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>50</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>50</height>
                        </size>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <layout class="QVBoxLayout" name="_HeaderLayout">
                        <property name="spacing">
                            <number>6</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QLabel" name="_Title">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>32</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>32</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="_Description">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="_Control" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>36</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>36</height>
                        </size>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="_ControlLayout">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QWidget" name="_TabPanel" native="true">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <layout class="QHBoxLayout" name="_TabPanelLayout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QPushButton" name="_TabPrev">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>16777215</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabUi">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                            <property name="checked">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabDownload">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabUpload">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabPlay">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabNetwork">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabOther">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabNext">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>16777215</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <spacer name="_ControlSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QStackedWidget" name="_Body">
                    <property name="currentIndex">
                        <number>4</number>
                    </property>
                    <widget class="RaptorUiPage" name="_UiPage"/>
                    <widget class="RaptorDownloadPage" name="_DownloadPage"/>
                    <widget class="RaptorUploadPage" name="_UploadPage"/>
                    <widget class="RaptorPlayPage" name="_PlayPage"/>
                    <widget class="RaptorNetworkPage" name="_NetworkPage"/>
                    <widget class="RaptorOtherPage" name="_OtherPage"/>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>RaptorUiPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Setting/Ui/RaptorUiPage.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>RaptorDownloadPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Setting/Download/RaptorDownloadPage.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>RaptorUploadPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Setting/Upload/RaptorUploadPage.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>RaptorPlayPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Setting/Play/RaptorPlayPage.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>RaptorNetworkPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Setting/Network/RaptorNetworkPage.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>RaptorOtherPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Setting/Other/RaptorOtherPage.h</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
